contracts/peterMangrove/MgvOracle/app.yml

```diff
@@ -0,0 +1,9 @@
+name: mangrove-oracle
+maintainer: peter@mangrove.exchange
+network: polygon-mumbai
+startBlock: 35665998
+version: 0.1.0
+contracts:
+  weth:
```

Comment:
You should probably change it to the contact name

Reply:
Good point, it has been changed